src/lib/helpers/toggleFunctions.js

```diff
@@ -0,0 +1,5 @@
+const toggleMode = (mode, setMode) => {
+	setMode(!mode);
+};
+
+export { toggleMode };
```

Comment:
This component handles logic that could typically be handled by the useState, you are passing, itself.